database-attachments.md

```diff
@@ -62,6 +62,14 @@ Alternatively, you can prepare a File model before hand, then manually associate
     $file->save();
 
     $model->avatar()->add($file);
+    
+Also you can attach the file to model by URL.
```

Comment:
Sentence should be "You can also add a file from a URL", make sure to add information on any extensions that are required in order for this method to work, i.e. allow_furl_open or the CURL extension